lib/datura/to_es/ead_to_es_items/fields.rb

```diff
@@ -203,16 +205,18 @@ def subjects
   end
 
   def subcategory
-    subcategory = get_text(@xpaths["subcategory"])
-    subcategory.length > 0 ? subcategory : "none"
+    # subcategory = get_text(@xpaths["subcategory"])
+    # subcategory.length > 0 ? subcategory : "none"
```

Comment:
This change doesn't seem to fit with the commit message so I don't know why it was done. Or maybe it was a temporary debugging change that got committed accidentally? What's the story here? :)

Reply:
I'm not sure why I commented out these lines, may have been accidentally committed. `subcategory` is not a field used by the new API, but this could help backward compatibility with the old API. I may have encountered an error related to `subcategory` when trying to post to the wrong API, and tried this solution.